pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
These instructions need to be updated the same way pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/create-an-ami/index.md was.
Different URL links, removing non-existent bundles, etc.